ts/components/services/ServiceMetadata/InformationRow.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { View, StyleSheet } from "react-native";
+import { View as NBView } from "native-base";
+
+import { TranslationKeys } from "../../../../locales/locales";
+import { capitalize } from "../../../utils/strings";
+import I18n from "../../../i18n";
+import { H4 } from "../../core/typography/H4";
+import TouchableDefaultOpacity from "../../TouchableDefaultOpacity";
+import ItemSeparatorComponent from "../../ItemSeparatorComponent";
+
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: "column"
+  },
+  touchable: {
+    flexDirection: "row",
+    marginVertical: 24
+  },
+  label: {
+    flex: 1
+  },
+  alignToRight: {
+    textAlign: "right"
+  }
+});
+
+type Props = {
+  value: string;
+  label: TranslationKeys;
+  onPress: () => void;
+  hint: TranslationKeys;
+};
+
+const InformationRow = ({ value, label, onPress, hint }: Props) => (
+  <View style={styles.row}>
+    <TouchableDefaultOpacity
+      onPress={onPress}
+      style={styles.touchable}
+      accessibilityRole={"button"}
+      accessibilityHint={I18n.t(hint)}
+    >
+      <H4
+        style={styles.label}
```

Comment:
```suggestion
```
by removing this `style`, the key-label will have a width corresponding to its exact content.
Now we have the row split in two parts: 50% key, 50% value
IMHO we have a lot of unused space that could be useful to show the value


| now  | with the proposed changes |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/126280837-b50fd073-262b-4db2-91b5-92540863f577.png) | ![after](https://user-images.githubusercontent.com/822471/126280844-3ad03119-331d-4d68-b1bf-e231295cc074.png)

@pietro909 and @thisisjp 
what do you think?

Reply:
I generally like the idea of having more space, but we should keep a minimum gutter between the columns, or we’ll “lose” the alignment on the right. What about 30%/70%?

Reply:
@thisisjp 
<img src="https://user-images.githubusercontent.com/822471/126281974-f459e7c5-c581-4c6c-8caa-71dd254ac66b.png" height="700"/>

```typescript
[...]
  label: {
    flex: 3
  },
  value: {
    flex: 7
  },
[...]
      <H4
        style={styles.label}
        ellipsizeMode={"tail"}
        numberOfLines={1}
        color={"bluegrey"}
        weight={"Regular"}
      >
        {capitalize(I18n.t(label))}
      </H4>
      <NBView hspacer={true} />
      <H4
        style={[styles.value, styles.alignToRight]}
        ellipsizeMode={"tail"}
        numberOfLines={1}
        color={"blue"}
        weight={"SemiBold"}
      >
        {value}
      </H4>
```

Reply:
I'm afraid we are bending the UI to a specific use case, that is where _content_left < content_right_.
This is but one of the possibilities, and frankly quite an edge case: phone numbers and fiscal codes are generally limited by regulation and emails are fairly short 'cause they need to be memory-friendly. 
The real outlier here is gonna be the `address` field here, which can be arbitrarily long. IMO there is no point in trying to manage it since we already open the maps upon press.

On top of that, what I believe we are not taking into account is that localization may backfire on the left-hand side items.

My opinion is that we should try to solve one single problem, that is **long sentences shall not break the UI**.  And a 50/50 enforcement solves it without making assumptions on which side will be more verbose.

Reply:
@pietro909 @Undermaken I think you both have a valid point:
1. Values used in this table template are usually short
2. Longer values either need a different layout, or you can just tap them to reveal\copy\use them
3. Labels are short by design, but we should consider localization issue

that said, if you think 50/50 is a safer go with it ;)

Reply:
@pietro909 if it's ok for you (50/50), I continue the review on the last code you committed